packages/html/src/drawer/tests/drawer-hierarchical-item-levels.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Icon } from '../../icon';
+
+const root = ReactDOM.createRoot(
+    document.getElementById('app') as HTMLElement
+);
+
+root.render(
+    <>
+        <div id="test-area" className="k-d-grid k-grid-cols-4">
+
+            <span>hierarchical items</span>
+            <span>hierarchical items - RTL</span>
+            <span></span>
+            <span></span>
+
+            <section>
+                <div className="k-drawer-container k-drawer-push k-drawer-expanded">
+                    <div className="k-widget k-drawer k-drawer-start">
+                        <div className="k-drawer-wrapper">
+                            <ul className="k-drawer-items">
+                                <li className="k-drawer-item k-level-0">
+                                    <Icon name="information" />
+                                    <span className="k-item-text">Getting Started</span>
+                                    <Icon name="arrow-chevron-down" className="k-drawer-toggle" />
+                                </li>
+                                <li className="k-drawer-item k-state-hover k-level-1">
+                                    <Icon name="question" />
+                                    <span className="k-item-text">About Kendo UI</span>
+                                </li>
+                                <li className="k-drawer-item k-state-selected k-level-2">
+                                    <Icon name="palette" />
+                                    <span className="k-item-text">Supported Themes</span>
+                                </li>
+                                <li className="k-drawer-item k-level-0">
+                                    <Icon name="zoom" />
+                                    <span className="k-item-text">Overview</span>
+                                    <Icon name="arrow-chevron-up" className="k-drawer-toggle" />
```

Comment:
Icons must not be clickable.

Add a wrapper around it with the class `k-drawer-toggle`

Reply:
We have discussed this internally with the Angular team, and currently there is no specific need to add the span wrapper at this point, as all events are attached on the list item and not on the icon itself.

As there is a little time until the code freeze, let's merge this PR as it is and if there is a need to wrap the Icon in a wrapper, we will do so in the future. 
@Raisolution , if i'm missing something, please feel free to fill the gaps.

cc: @Juveniel